config/locales/es.yml

```
@@ -1,6 +1,6 @@
 ---
 es:
-  credentials_required:
+  credentials_required: Faltan credenciales
```

```suggestion
  credentials_required: Credenciales requeridas
```
No faltan, se requiren.